compiler-rt/test/profile/Linux/coverage-debug-info-correlate.cpp

```diff
@@ -48,7 +48,7 @@
 
 // RUN: diff <(llvm-profdata show --all-functions --counts %t.normal.profdata) <(llvm-profdata show --all-functions --counts %t.profdata)
 
-// RUN: llvm-cov export --format=lcov --instr-profile=%t.profdata %t | FileCheck %s -check-prefix=NAME
+// RUN: llvm-cov export --format=lcov --instr-profile=%t.profdata %t --debug-info=%t | FileCheck %s -check-prefix=NAME
```

Comment:
If the binary already has debug info, we could use for correlation instead of passing the redundant `--debug-info=%t`, right? Of course this wouldn't work for MachO or if the binary is stripped of debug info. Is it possible (or at least easy to only require `--debug-info` if the provided binary doesn't already have it?